apis/v1alpha1/backendpolicy_types.go

```diff
@@ -31,16 +31,25 @@ import (
 // implementation specific may be represented with similar implementation
 // specific custom resources.
 type BackendPolicy struct {
-	metav1.TypeMeta   `json:",inline"`
+	metav1.TypeMeta `json:",inline"`
+	// +optional
```

Comment:
why optional here?

Reply:
@jpeach I'm following k8s api conventions. When a field includes `omitempty`, it should be tagged +optional.

xref: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#optional-vs-required
xref: https://github.com/kubernetes/api/blob/master/apps/v1/types.go

Reply:
Is it that the `omitempty` tag here is wrong instead?
How can `metav1.ObjectMeta` possibly be empty for any valid k8s resource?